apps/costcontrol/elements/datausage-tab.html

```diff
@@ -16,20 +16,20 @@
 
         <ul class="toggle-data" data-type="list">
           <li id="mobileItem">
-            <gaia-checkbox id="mobileCheck">
-              <label>
-                <span data-l10n-id="mobile-usage">Mobile usage</span>
-                <span class="overview" id="mobileOverview"></span>
-              </label>
-            </gaia-checkbox>
+            <label class="pack-checkbox">
```

Comment:
OK, if it's the best way to resolve this issue, I agree, but please, open a follow-up to undone the change when the gaia-checkbox component supports RTL and put a reference to the bug for tracking it (I suppose exists a bug to indicate that component doesn't support RTL if not, please, file a bug for this)